spec/actors/curation_concerns/file_set_actor_spec.rb

```diff
@@ -141,4 +141,28 @@
       end
     end
   end
+
+  describe "#attach_file_to_work" do
+    let(:file_set2) { create(:file_set) }
+    let(:file_set3) { create(:file_set) }
+    let!(:work_v1) do
+      work = create(:generic_work)
+      # this is not part of a block on the create, since the work must be saved be fore the representative can be assigned
+      work.members << file_set
+      work.save
+      work
+    end
+
+    let!(:work_v2) do
+      work = ActiveFedora::Base.find(work_v1.id)
+      work.members << file_set2
+      work.save
+      work
+    end
+
+    it "writes to the most up to date version" do
+      actor.send(:attach_file_to_work, work_v1, file_set3, {})
```

Comment:
Can you add a comment here that `#attach_file_to_work` is private, and that's why you're invoking via `#send`?